src/mem/cache/tags/tagged_entry.hh

```diff
@@ -114,44 +102,34 @@ class TaggedEntry : public ReplaceableEntry
             isSecure(), isValid(), ReplaceableEntry::print());
     }
 
-  protected:
-    /**
-     * Set tag associated to this block.
-     *
-     * @param tag The tag value.
-     */
-    virtual void setTag(Addr tag) { _tag = tag; }
+    bool
+    matchTag(const Addr tag) const override {
+        panic("Need is_secure arg");
+        return false;
+    }
```

Comment:
Minor coding style, this should be

bool
matchTag(const Addr tag) const override
{
}

Reply:
Updated.